src/core/client/stream/tabs/Comments/Comment/MarkCommentAsSeenMutation.tsx

```diff
@@ -9,18 +9,15 @@ import {
 
 import {
   MarkCommentAsSeenMutation,
-  MarkCommentSeenInput,
+  MarkCommentsAsSeenInput,
 } from "coral-stream/__generated__/MarkCommentAsSeenMutation.graphql";
 
-import {
-  COMMIT_SEEN_EVENT,
-  CommitSeenEventData,
-} from "../commentSeen/CommentSeenContext";
+import { COMMIT_SEEN_EVENT, CommitSeenEventData } from "../commentSeen/";
 
 const mutation = graphql`
-  mutation MarkCommentAsSeenMutation($input: MarkCommentSeenInput!) {
-    markCommentAsSeen(input: $input) {
-      comment {
+  mutation MarkCommentAsSeenMutation($input: MarkCommentsAsSeenInput!) {
```

Comment:
Maybe rename the mutation as well?

Reply:
Yep yep, didn't realize my code-wide rename was missing this. It's done now.